### ISSUE TITLE
Problem: AllowedCLIIP config implemention throws errors

### ISSUE THREAD

**Issue by argltuc** [FreeTAKTeam/FreeTakServer#406]
When setup FreeTakServer (latest master commit [56ea1f6](https://github.com/FreeTAKTeam/FreeTakServer/commit/56ea1f647b79bd79a5c586730f2bca106a767f80)) by executing `pip install -e ./` from project root, resulting FTS service module `FreeTAKServer.controllers.services.FTS` is not runable due to missing regular expression module:

```
"FreeTAKServer\FreeTAKServer\controllers\configuration\MainConfig.py", line 183, in MainConfig
    AllowedCLIIPs = re.split(r'[,:]', os.environ.get("FTS_CLI_WHITELIST")) or AllowedCLIIPs
NameError: name 're' is not defined
```

After importing re module, implemention of splitting enironment variable FTS_CLI_WHITELIST throws errors, if environment variable is not set on host.

Related lines in sources:
https://github.com/FreeTAKTeam/FreeTakServer/blob/56ea1f647b79bd79a5c586730f2bca106a767f80/FreeTAKServer/controllers/configuration/MainConfig.py#L183

**Comment by brothercorvo** [comment]
@dlc-ariel I think we removed the package because of 3.11?

**Comment by dlc-ariel** [comment]
@brothercorvo No, this package is still a requirement. This issue is already being addressed by @hickey  in #386. We can wait for the 386 to be complete or accept this pr

**Comment by hickey** [comment]
I have moved #386 out of draft as I have resolved all the lingering issues. I would somewhat like to wait until #400 is complete and merged so that I can rebase #386 and see that the testing gets better and let #386 go through a full lint. I just commented on #400 and think that it is ready once the one test failing linting is comment out (see my last comment on that issue).

**Comment by argltuc** [comment]
okay, i didn't recognize the other PR related to this problem.
Since it seems a more complex problem instead just a quick hotfix, it would be preferable to follow #400 and #386 and close my related PR #407 but leave this issue open for reference until #386 is merged.
